shared/constants/types/fs.js

```diff
@@ -75,4 +76,5 @@ export const stringToPathType = (s: string): PathType => {
   }
 }
 export const pathTypeToString = (p: PathType): string => p
-export const pathConcat = (p: Path, s: string): Path => stringToPath(pathToString(p) + '/' + s)
+export const pathConcat = (p: Path, s: string): Path =>
```

Comment:
There's a `join` function in `path.posix` which handles the stuff like consolidating `/`s etc. Is there a reason not to use that?

Reply:
I asked earlier about any path libraries we were using and I was told to implement my own. And it was simple enough. So I did.

Reply:
Also here we don't really want posix path behavior. We want it to always be `/`.

Reply:
`path.posix` in node is always the unix style I think, if that's what you mean. They have `path.win32` for Windows, and `path` is auto-detected based on OS.

Reply:
I see. Well I didn't test whether it's available in RN, and it was easy to implement our homegrown version.